extensions/actions/AnswerSubmit/AnswerSubmit.py

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Python configuration for recording the action when a learner submits an
```

Comment:
when a learner submits an answer --> corresponding to a learner submitting an answer

Similar changes below.

Reply:
Done